Source/ablastr/fields/IntegratedGreenFunctionSolver.cpp

```diff
@@ -124,17 +133,10 @@ computePhiIGF ( amrex::MultiFab const & rho,
     tmp_rho.ParallelCopy( rho, 0, 0, 1, amrex::IntVect::TheZeroVector(), amrex::IntVect::TheZeroVector() );
     BL_PROFILE_VAR_STOP(timer_pcopies);
 
-#if !defined(ABLASTR_USE_HEFFTE)
-    // Without distributed FFTs (i.e. without heFFTe):
-    // We loop over the original box (not the 2x wider one), and the other quadrants by periodicity
-    amrex::BoxArray const& igf_compute_box = amrex::BoxArray( domain );
-#else
-    // With distributed FFTs (i.e. with heFFTe):
-    // We loop over the full 2x wider box, since 1 MPI rank does not necessarily own the data for the other quadrants
-    amrex::BoxArray const& igf_compute_box = tmp_G.boxArray();
-#endif
+    amrex::BoxArray const& igf_compute_box = (is_2d_slices || is_3d_distributed) ? tmp_G.boxArray() : amrex::BoxArray(domain);
```

Comment:
Could you reintroduce some of the comments (or modified versions thereof) to explain what this line does?
(People reading this code might not be sure what its purpose is.)